cypress/integration/tests/developer/applicationinventory/analysis/applicationcrud.test.ts

```diff
@@ -0,0 +1,87 @@
+/// <reference types="cypress" />
+
+import {
+    exists,
+    hasToBeSkipped,
+    login,
+    preservecookies,
+    deleteAllStakeholders,
+    createMultipleBusinessServices,
+    deleteAllBusinessServices,
+    notExists,
+} from "../../../../../utils/utils";
+
+import { ApplicationInventory } from "../../../../models/developer/applicationinventory/applicationinventory";
+
+import * as data from "../../../../../utils/data_utils";
+import { Stakeholders } from "../../../../models/developer/controls/stakeholders";
+import { BusinessServices } from "../../../../models/developer/controls/businessservices";
+
+import { publicRepo } from "./analysis_config";
+
+var stakeholdersList: Array<Stakeholders> = [];
+var stakeholdersNameList: Array<string> = [];
+var businessservicesList: Array<BusinessServices> = [];
+
+describe("Application crud operations from the Analysis tab", { tags: "@tier1" }, () => {
```

Comment:
I think application crud operations can be under assessment folder . Under analysis we will cover different type of analysis.